### ISSUE TITLE
Fix map.view.center.xyz channel descriptions

### ISSUE THREAD

**Issue by cbashioum** [CMAPI/cmapi#67]
From Pigelvy to the cmapi mailing list:
*************
The documentation of map.view.center doesn’t look good to me.

For example, on the left side tree one can see map.view.overlay, map.view.feature, map.view.location, map.view.bounds but when one clicks on them, on the right side the title says map.view.center.overlay/map.view.center.feature/map.view.center.location/map.view.center.bounds

Shouldn’t the left menu dealing with map.view.center messages be hierarchically ordered?
***********
Basically, the tree view should be set to map.view.center, and underneath that should be map.view.center.overlay, map.view.center.feature, map.view.center.location, and map.view.center.bounds

**Comment by cbashioum** [comment]
Fixed but not yet pushed to the site

**Comment by cbashioum** [comment]
Eric -please push to public
